jina/serve/runtimes/asyncio.py

```diff
@@ -65,7 +65,13 @@ def __init__(
             )
 
         self._setup_monitoring()
-        send_telemetry_event(event='start', obj=self)
+        flow_id = self._flow_id
+        telemetry_kwargs = {}
+
+        if flow_id is not None:
+            telemetry_kwargs = {'flow_id': flow_id}
+        send_telemetry_event(event='start', obj=self, **telemetry_kwargs)
+        self._start_time = time.time()
```

Comment:
I think it's better to have function `_get_telemetry_extra_kwargs` and override it in gateway runtime to flow id.
I don't like much how we set a `None` flow id for all pods.
I'm also okay with renaming to `pod_id` and then each runtime can implement it on his own

Reply:
We do not sent a `None` flow_id, we just do not send it